docs/UserManuals/ConfigUI/GitLab.md

```diff
@@ -49,3 +49,28 @@ There are no transformation rules for GitLab repos.
 
 ### Step 4 - Setting Sync Frequency
 You can choose how often you would like to sync your data in this step by selecting a sync frequency option or enter a cron code to specify your prefered schedule.
+
+
+## FAQ
+
+### How to collect data from Gitlab server with Self-Signed Certificate
+There are two symptoms would exhibit when trying to collect data from a private Gitlab server with a Self-Signed certificate:
```

Comment:
There might be two problems when trying to collect data from a private GitLab server with a self-signed certificate: